src/ui-test/test-data/fuse/blueprint-url-2.21.0.fuse-730078-redhat-00001.json

```diff
@@ -0,0 +1,5 @@
+{
+    "framework": "blueprint",
+    "type": "url",
+    "camelVersion": "2.21.0.fuse-730078-redhat-00001"
```

Comment:
maybe we should use the latest version available which should be a 7.5.0. What do you think?

Reply:
Thank you for pointing out. I will definitely change it. It should be 2.21.0.fuse-750033-redhat-00001, should not it?